content/reference/user/package_gen/index.md

```diff
@@ -1,23 +1,23 @@
 ---
-title: "Artifact Generation"
-description: "Learn how to generate an artifact for Gatling Enterprise from the Gatling zip bundle, or from a Maven, SBT, or Gradle project."
-lead: "Generate an artifact from your Gatling bundle or with a Maven, SBT, or Gradle project."
+title: "Package Generation"
+description: "Learn how to generate a package for Gatling Enterprise from the Gatling zip bundle, or from a Maven, SBT, or Gradle project."
+lead: "Generate a package from your Gatling bundle or with a Maven, SBT, or Gradle project."
 date: 2021-03-08T12:50:32+00:00
 lastmod: 2021-08-05T13:13:30+00:00
 weight: 10040
```

Comment:
add an alias to redirect from artifact_gen to package_gen

Reply:
You're right, I planned to do that but eventually forgot